internal/util/vault.go

```diff
@@ -244,6 +285,9 @@ Vault will verify service account jwt token with Kubernetes and return token
 if the requester is allowed.
 */
 func (kms *VaultKMS) getAccessToken() (string, error) {
+	if kms.VaultToken != "" {
+		return kms.VaultToken, nil
```

Comment:
Does `kms.VaultToken` not expire?

Reply:
It depends on how it is setup in Vault, the admin can decide to renew it or have it never expire.